.gitignore

```diff
@@ -0,0 +1,6 @@
+﻿################################################################################
```

Comment:
Here is cool service https://www.gitignore.io/. It helps to generate good gitignore files for your IDE. Please use is and update this file.